pom.xml

```diff
@@ -1,12 +1,4 @@
 <project>
-  <modelVersion>4.0.0</modelVersion>
```

Comment:
The tests seem to use this pom as an example.

Generally ordering isn't stable with dry-struct, they're stored in a Hash (imagine a `LinkedHashMap` where the insert order is maintained)